__tests__/developers.test.js

```diff
@@ -110,5 +97,30 @@ describe('Developers', () => {
       const res = await getWord(limitWordId, { apiLimit: 2 }, { apiKey: developerRes.body.apiKey });
       expect(res.status).toEqual(403);
     });
+
+    it('should return developer document with correct credentials', async () => {
+      const developerRes = await createDeveloper(developerData);
+      const developerDetails = await getDeveloper({ apiKey: developerRes.body.apiKey });
+      expect(developerDetails.status).toEqual(200);
+      expect(developerDetails.body.developer).toMatchObject({
+        usage: expect.objectContaining({
+          date: expect.any(String),
+          count: expect.any(Number),
+        }),
+        name: expect.any(String),
+        apiKey: expect.any(String),
+        email: expect.any(String),
+        password: expect.any(String),
+        createdAt: expect.any(String),
+        updatedAt: expect.any(String),
+        id: expect.any(String),
```

Comment:
you want to expect the specific data that you provided to create developer - specifically you want to check to make sure the usage.count is 0, name and email is what you provided